pyproject.toml

```diff
@@ -1,23 +1,20 @@
 [build-system]
-requires = ["setuptools >= 40.6.0", "setuptools_scm[toml] >= 4", "cython"]
 build-backend = "setuptools.build_meta"
+requires = ["setuptools >= 40.6.0", "setuptools_scm[toml] >= 4", "cython"]
```

Comment:
looks like these were sorted, which is fine by me

Reply:
yep, looks like one of the VS Code extensions is formatting this file on save

Reply:
honestly, if it'd be just a line it'd be fine but if every developer with different settings keeps changing files around it's going to be a mess…
Can we avoid that?
I hate to be the grumpy guy sorry 😢

Reply:
Unless we enforce linting/formatting on toml files to ensure everyone is consistent, let's revert the sorting change.

(is there a vscode settings file we can add to the repo to disable that, or... something to ensure a consistent experience for vscode users?)

Reply:
I have just added
~~~ json
    "[toml]": {
        "editor.formatOnSave": false
    },
~~~
to my vscode settings